Changelog.md

```diff
@@ -8,6 +8,7 @@
 - Update Camel Quarkus Catalog from 1.8.1 to 2.0.0
 - Adapt for Camel K community 1.5/Camel K Red Hat 1.4
   - Provide completion for new Camel K modeline option names
+  - Provide completion the different kind for `config` Camel K modeline option (`configmap`, `secret`, `file`) 

```

Comment:
completion "for" ?

Reply:
I think it should be:
"Provide completion for the different kinds of `config` Camel K modeline option ..."

What do you think?

Reply:
I updated to almost that: https://github.com/camel-tooling/camel-lsp-client-vscode/pull/666/files#diff-c275fec9c453c2a42515bc5ab47e30fa4130ff1426aef2f6e000a9a34e122cb8R11